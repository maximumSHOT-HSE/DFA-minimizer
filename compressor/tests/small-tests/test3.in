5 10 2
1 2 0
1 4 1
2 4 0
2 5 1
3 2 0
3 4 1
4 2 0
4 1 1
5 2 0
5 1 1
2
5
2 5 2 1 5 
