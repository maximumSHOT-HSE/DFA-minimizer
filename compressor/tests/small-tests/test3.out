5 10 2
4 1 1
5 1 1
1 2 0
3 2 0
4 2 0
5 2 0
2 4 0
1 4 1
3 4 1
2 5 1
2
3
1 2 5 
